readthedocs/comments/models.py

```diff
@@ -111,6 +111,7 @@ class DocumentNodeSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = DocumentNode
+        exclude = ('')
```

Comment:
Looking at http://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include isn't this actually an equivalent of `fields = '__all__'`?